Common++/src/GeneralUtils.cpp

```diff
@@ -1,7 +1,7 @@
 #define LOG_MODULE CommonLogModuleGenericUtils
 
-#include "GeneralUtils.h"
-#include "Logger.h"
+#include "pcapplusplus/GeneralUtils.h"
+#include "pcapplusplus/Logger.h"
```

Comment:
Include paths are sometimes case sensitive depending on the platform. The renamed folder is camel case naming convention while this is all lowercase.

Reply:
Thanks and fuck APFS and there case insensitive bullshit!

Reply:
Yeah, that is why I have read that it is better to use only lowercase for files, so you don't have the issue of some systems being case sensitive and some being insensitive.